apps/api/src/domain/models/term/repositories/arango-term-query-repository.spec.ts

```diff
@@ -542,4 +544,45 @@ describe(`ArangoTermQueryRepository`, () => {
             );
         });
     });
+
+    describe(`indexVocabularyList`, () => {
+        describe(`when the vocabulary list exists`, () => {
+            const vocabularyListId = buildDummyUuid(596);
+
+            const vocabularyListName = buildMultilingualTextFromBilingualText(
+                { text: 'vl name', languageCode: LanguageCode.Chilcotin },
+                { text: 'vl name (translated to English', languageCode: LanguageCode.English }
+            );
+
+            const targetTerm = termViews[0];
+
+            beforeEach(async () => {
+                await databaseProvider.clearViews();
+
+                await databaseProvider.getDatabaseForCollection('contributors').clear();
+
+                await testQueryRepository.create(targetTerm);
+
+                // await testQueryRepository.create(vocabularyListView)
+            });
+
+            it(`should update the view`, async () => {
```

Comment:
So, do I understand correctly that for a given term, `indexVocabularyList` will retrieve the vocabulary lists it exists in?  If so, just wondering if this it statement could read `should update the view with the vocabulary lists associated with this term` or something like that?

Reply:
Yea I agree. My wording is vague. Maybe, "should update the term's view with the vocabulary list it is contained in".

Reply:
Done.